modules/react/breadcrumbs/lib/BreadcrumbsItem.tsx

```diff
@@ -38,7 +38,7 @@ export const useBreadcrumbsItem = composeHooks(
   createElemPropsHook(useBreadcrumbsModel)(
     (
       {state},
-      _?: React.Ref<HTMLElement>,
+      _,
```

Comment:
Question: is moving this from optional to required a breaking change? Do we need to help consumers update?